library/src/org/wordpress/passcodelock/AppLockManager.java

```diff
@@ -3,35 +3,32 @@
 import android.app.Application;
 
 public class AppLockManager {
-
     private static AppLockManager instance;
     private AbstractAppLock currentAppLocker;
-    
+
     public static AppLockManager getInstance() {
         if (instance == null) {
             instance = new AppLockManager();
         }
         return instance;
     }
-        
+
     public void enableDefaultAppLockIfAvailable(Application currentApp) {
-        if (android.os.Build.VERSION.SDK_INT >= 14) {
-            currentAppLocker = new DefaultAppLock(currentApp);
-            currentAppLocker.enable();
-        }
+        if (!DefaultAppLock.isSupportedApi()) return;
+        currentAppLocker = new DefaultAppLock(currentApp);
+        currentAppLocker.enable();
+    }
+
+    public boolean isDefaultLock() {
+        return getAppLock() != null && getAppLock() instanceof DefaultAppLock;
     }
     
     /**
      * Default App lock is available on Android-v14 or higher.
```

Comment:
Niptick: I think we should change the comment in something like:
`True when an App lock is available. It could be either a the Default App lock on Android-v14 or higher, or a non default App lock`.